roles/edpm_network_config/tasks/dns_nm_configs.yml

```diff
@@ -42,7 +42,7 @@
       loop:
         - /etc/NetworkManager/NetworkManager.conf
         - /etc/NetworkManager/conf.d/99-cloud-init.conf
-    - name: Set 'rc-manager=unmanaged' in /etc/NetworkManager/NetworkManager.conf
+    - name: Unset 'rc-manager=unmanaged' in /etc/NetworkManager/NetworkManager.conf
```

Comment:
Below Reload task can also be made conditional i.e no need to reload if desired config is in place